keras/layers/preprocessing/text_vectorization.py

```diff
@@ -513,7 +513,7 @@ def call(self, inputs):
     lookup_data = self._lookup_layer(inputs)
 
     # For any non-int output, we can return directly from the underlying layer.
-    if self._output_mode is not INT:
+    if self._output_mode != INT:
```

Comment:
Both are equivalent (the objects being compared aren't pointers)

Reply:
see this issue: https://github.com/keras-team/keras/issues/15382